shared/settings/notifications/index.js

```diff
@@ -91,6 +94,19 @@ const Notifications = (props: Props) =>
             unsubscribedFromAll={props.groups.app_push.unsubscribedFromAll}
           />}
 
+        {props.groups.security &&
+          props.groups.security.settings &&
+          <Group
+            allowEdit={props.allowEdit}
+            groupName="security"
+            onToggle={props.onToggle}
+            onToggleUnsubscribeAll={() => {}}
```

Comment:
You sure? I think it is cleaner like this personally. I feel like all the null checks just make the code more verbose. This way the component can just call the function (which can never happen), and nothing happens.